src/widget/wtrackmenu.cpp

```diff
@@ -151,6 +153,11 @@ void WTrackMenu::createMenus() {
         m_pBPMMenu->setTitle(tr("Adjust BPM"));
     }
 
+    if (featureIsEnabled(Feature::Properties)) {
+        m_pRatingMenu = new QMenu(this);
+        m_pRatingMenu->setTitle(tr("Set Rating"));
```

Comment:
I'm not sure why the title is set in a separate step, not in the initializer.
So I simply copied how it's done everywhere else.